admin/client/App/sagas/index.js

```diff
@@ -8,8 +8,9 @@ import { loadItems } from '../screens/List/actions';
 /**
  * Load the items
```

Comment:
That sounds reasonable to me, what about `filtersToParams` or something like that?